### ISSUE TITLE
ios-communication-library 0.94.0, version solving failed

### ISSUE THREAD

**Issue by rahioza** [nextcloud/ios#1574]
The app building fails showing versioning error like attached screenshot. 
![Screen Shot 2021-04-19 at 2 57 44 PM](https://user-images.githubusercontent.com/82640242/115214915-9a9df280-a120-11eb-9db4-405b3284f90c.png)
what changes needed to resolve this?

**Comment by marinofaggiana** [comment]
hi @rahioza which branch you used ?

**Comment by rahioza** [comment]
Hello,

        Thank you for your response. I really appreciate it. I have used
the master branch.

Thanks,
Rahi Oza

**Comment by marinofaggiana** [comment]
Yes, sorry for delay, can you re-try with the new version 3.4.2 ?

**Comment by rahioza** [comment]
I tried with that version 3.4.2 and now it is showing new versioning error like below

![Screen Shot 2021-04-29 at 5 29 08 PM](https://user-images.githubusercontent.com/82640242/116548350-b7e97280-a911-11eb-94dc-3c4f29219a84.png)

**Comment by rahioza** [comment]
![Screen Shot 2021-04-29 at 5 29 08 PM](https://user-images.githubusercontent.com/82640242/116548528-faab4a80-a911-11eb-919f-5cce7842afce.png)
